### DISCUSSION TITLE
Debugging up-animation

### DISCUSSION THREAD

**Discussion by s5no5t** [unpoly/unpoly#457] [Q&A]
Hi everyone,

I'm experimenting with unpoly and got stuck with a behavior I can't see the reason for.

I have a  link in my app that opens a new overlay. It uses some appear animation by default.

However, when I apply a different animation,  e.g. `up-animation="move-from-top"`, the animation doesn't change.

How do I find the reason that unpoly doesn't pick up the attribute here?

Here is the debug output I have
![image](https://user-images.githubusercontent.com/3235163/221527420-06cf1224-befa-4a8d-b5a6-e917fbef5603.png)

**Comment by adam12** [comment]
Can you confirm which version of Unpoly you are using? And for posterity, can you show the markup/code you're using to define which animation to be used?

**Comment by triskweline** [comment]
This is a bug in Unpoly. It is fixed in 2.7.2 (released just now).
